plugins/CoreHome/angularjs/report-export/reportexport.directive.js

```diff
@@ -185,10 +185,10 @@
                     scope.reportType          = 'default';
                     scope.reportLimit         = dataTable.param.filter_limit > 0 ? dataTable.param.filter_limit : 100;
                     scope.reportLimitAll      = dataTable.param.filter_limit == -1 ? 'yes' : 'no';
-                    scope.optionFlat          = dataTable.param.flat;
+                    scope.optionFlat          = dataTable.param.flat === true || dataTable.param.flat === 1 || dataTable.param.flat === "1";
```

Comment:
in some cases `dataTable.param.flat` is set to (string) `0`, which validated to true before, causing a `flat=1` in the export url where it shouldn't be